.changeset/deep-sloths-fetch.md

```diff
@@ -0,0 +1,21 @@
+---
+"@spectrum-css/actionbar": major
+---
+
+### Actionbar S2 Migration
+
+Action bar now has some updated colors, corner radius, icons and outline has been removed. The default and emphasized variant now have a light and dark mode theme.
```

Comment:
Could we clarify the part about the outline/border a little? The border is still there, but just different colors based on light vs dark modes.